chrome.tera

```diff
@@ -0,0 +1,48 @@
+---
+whiskers:
+  version: 2.0.0
+  matrix:
+    - flavor
+    - accent
+  filename: "themes/{{ flavor.identifier }}/{{ accent }}/manifest.json"
+version: 4.1
+---
+{%- set palette = flavor.colors -%}
+{%- set identCap = flavor.identifier | capitalize -%}
+{%- set accentCap = accent | capitalize -%}
+{%- macro rgb(c) -%}
+[{{c.rgb.r}}, {{c.rgb.g}}, {{c.rgb.b}}]
+{%- endmacro rgb -%}
```

Comment:
```suggestion
{%- macro rgb(color) -%}
[{{ color.rgb.r }}, {{ color.rgb.g }}, {{ color.rgb.b }}]
{%- endmacro rgb -%}
```

Reply:
Reverted this because it makes the invocations much longer and more cluttered than they have to be. Remember macros have no positional arguments, only keyword arguments!

Reply:
Yeah. One of my main gripes with Tera...